sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3189,6 +3189,14 @@ object SQLConf {
       .version("4.0.0")
       .fallbackConf(Python.PYTHON_WORKER_FAULTHANLDER_ENABLED)
 
+  val PYTHON_DATA_SOURCE_STATIC_IMPORT_ENABLED =
+    buildConf("spark.sql.pyspark.dataSource.staticImport.enabled")
```

Comment:
I think this should be a `StaticSQLConf` because loading Python Data Source should happen once statically for all sessions.